web/screenshotTesting/app.screenshot-spec.ts

```diff
@@ -0,0 +1,12 @@
+import { AppPage } from './app.po';
+import { $, browser, logging } from 'protractor';
+
+describe('protractor-image-comparison desktop', () => {
+  beforeEach(async () => {
+    await browser.get(browser.baseUrl);
```

Comment:
Shouldn't this be inside the `it`? I presume we'll want to visit different pages for different tests.